README.md

```diff
@@ -538,7 +512,7 @@ and maximum values, we can define a simple predicate saying whether a
 given input vector is in the right range:
 
 ``` vehicle
-validInput : UnnormalisedInputVector -> Bool
+validInput : UnnormalisedInput -> Bool
 validInput x = forall i .
   minimumInputValues ! i <= x ! i <= maximumInputValues ! i
```

Comment:
I think this could now be written as `minimumInputValues <= x <= maximumInputValues`?